web/public/locales/sv.json

```diff
@@ -1850,6 +1857,14 @@
     },
     "ZW": {
       "zoneName": "Zimbabwe"
+    },
+
+    "US": {
+      "zoneName": "Amerikas f�renta stater"
```

Comment:
Is it just Github that cannot show the `ö` or is something else going on? 🤔

Reply:
Strange, I'll look into if the coding changed on the file. Or if there is something else going on.

Reply:
I think I found the issue, it's neither the coding or github. It's the translation helper...

I'll update the translations manually and then look into that script to see what's going on under the hood.